refinery/ui/source/js/user-file-browser/services/factory.js

```diff
@@ -21,7 +21,8 @@
     return {
       getUserFiles: getUserFiles,
```

Comment:
besides alphabetization, what should change? Am I violating single-responsibility?